src/app/(BottomTabNavigation)/Cases/QRCodeScanner/AddCase/index.tsx

```diff
@@ -0,0 +1,75 @@
+import { useLocalSearchParams, useRouter } from 'expo-router';
+import React, { useEffect, useState } from 'react';
+import { Image, Text, TouchableOpacity, View } from 'react-native';
+import { getCase, uploadCase } from '../../../../../supabase/queries/case';
+import styles from './styles';
+
+enum caseStatus {
```

Comment:
We don't need `CaseStatus` or `Case` defined in here because it's in the `types` directory!